### PR TITLE
Fix text in help panel.

### PR DESCRIPTION
**Pull request by danielfenner**

### CODE REVIEW

File: OrbitGl/CaptureWindow.cpp [hunk 1]
```diff
@@ -924,10 +924,11 @@ void CaptureWindow::RenderHelpUi() {
     return;
   }
 
-  ImGui::Text("Start/Stop capture: 'X'");
-  ImGui::Text("Time zoom: scroll or CTRL+right-click/drag");
+  ImGui::Text("Start/Stop capture: 'x'");
+  ImGui::Text("Pan/Zoom in capture: 'a','s','w','d'");
```
Comment (ronaldfw) [inline OrbitGl/CaptureWindow.cpp:928]: Would switch order to WASD, as that is what most people will be familiar with (mostly from games).

File: OrbitGl/CaptureWindow.cpp [hunk 2]
```diff
@@ -924,10 +924,11 @@ void CaptureWindow::RenderHelpUi() {
...
+  ImGui::Text("Left click + drag to select or pan");
   ImGui::Text("Y axis zoom: CTRL+scroll");
-  ImGui::Text("Zoom last 2 seconds: 'A'");
+  ImGui::Text("Right click + drag to measure time");
   ImGui::Separator();
```
Comment (ronaldfw) [inline OrbitGl/CaptureWindow.cpp:932]: CTRL+right-click + drag for zoom is really useful, so it'd be great to also mention here (it's deleted above).

File: OrbitGl/CaptureWindow.cpp (hunk 1 above)
Comment (danielfenner) [reply OrbitGl/CaptureWindow.cpp]: done

File: OrbitGl/CaptureWindow.cpp (hunk 2 above)
Comment (danielfenner) [reply OrbitGl/CaptureWindow.cpp:933]: Thanks, that got lost ...

**Pull request merged**